xarray/test/test_dask.py

```diff
@@ -300,3 +300,35 @@ def test_stack(self):
                              dims=['w', 'z'])
         assert stacked.data.chunks == expected.data.chunks
         self.assertLazyAndIdentical(expected, stacked)
+    
+    def test_dot(self):
+        
+        eager = self.eager_array.dot(self.eager_array[0])
+        lazy = self.lazy_array.dot(self.lazy_array[0])
+        self.assertLazyAndAllClose(eager, lazy) 
+        
+        x_trans = np.linspace(-3,3,6)
```

Comment:
I would probably drop these test cases -- they are already tested in your other test.